src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py

```diff
@@ -68,7 +70,9 @@ def split(self):
                                  NumberOfLumis = nLumis,
                                  NumberOfEvents = nEvents,
                                  Jobs = jobs,
-                                 Mask = copy(mask))
+                                 Mask = copy(mask),
+                                 BlowupFactor = self.args['BlowupFactor']))
```

Comment:
It's failing in jenkins because the code is breaking here (additional closing parenthesis).